.github/actions/auto-assign-project.yml

```diff
@@ -0,0 +1,32 @@
+name: Assign to One Project
+description: 'Rules for auto-assigning projects'
+
+on:
+  issues:
+    types: [opened, labeled]
+  pull_request:
+    types: [opened, labeled]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
+    steps:
+    - name: Assign NEW issues to Backlog
+      uses: srggrs/assign-one-project-github-action@1.2.1
+      if: |
+        github.event.action == 'opened' &&
+        github.event == 'issue'
+      with:
+        project: 'https://github.com/CityOfDetroit/affordable-housing-app/projects/2'
+
+#    - name: Assign issues and pull requests with `bug` label to project 3
```

Comment:
Not sure how nitty to be but should we remove the comments?

Reply:
We can, I left it in there as an example of how we might extend it in the future, but I suppose we could always just look up the examples on the action page. WDYT?